packages/client/hmi-client/src/page/project/components/tera-project-configuration-modal.vue

```diff
@@ -111,20 +112,19 @@ const props = defineProps<{
 
 const emit = defineEmits(['close-modal']);
 
-const auth = useAuthStore();
 const router = useRouter();
-const userId = auth.user?.id ?? '';
 
 const title = ref(props.project?.name ?? '');
+const isTitleInvalid = computed(() => isEmpty(title.value));
 const description = ref(props.project?.description ?? '');
 const isApplyingConfiguration = ref(false);
 
-async function createProject() {
+async function createProject(name: string, desc: string) {
 	isApplyingConfiguration.value = true;
-	const project = await useProjects().create(title.value, description.value, userId);
+	const project = await useProjects().create(name, desc);
```

Comment:
We have the back end returning a `400` for a project without a name which is perfect, but, do we want to disable the 'ok' button or check for an empty name here to save us a 400 just in case (and have a nicer looking error as a result, or the inabillity to even hit OK?)

Reply:
I agree, but I think revamping the whole front-end for that was a bit too much for that simple PR.